tests/framework/PostProcessors/InterfacedPostProcessor/RiskMeasuresTimeDep/testPrintHistorySet_dump_0.csv

```diff
@@ -3,7 +3,8 @@ Astatus,Bstatus,time
 1,1,0.5
 1,1,1
 1,1,1.5
-1,1,21,1,2.5
```

Comment:
Oops! I see the problem you fixed here. There was a missing new line. I approve this change.